client/app/scripts/selectors/nodes-chart-graph.js

```diff
@@ -3,7 +3,7 @@ import { createSelector, createStructuredSelector } from 'reselect';
 import { Map as makeMap } from 'immutable';
 import timely from 'timely';
 
-import { initEdgesFromNodes, collapseMultiEdges } from '../utils/layouter-utils';
+import { initEdgesFromNodes } from '../utils/layouter-utils';
```

Comment:
Since `collapseMultiEdges` is not gonna be used anymore, I'd just remove it from `layouter-utils`, together with the test.